docs/data/base/components/fieldset/fieldset.md

```diff
@@ -0,0 +1,78 @@
+---
+productId: base-ui
+title: React Fieldset component and hook
+components: FieldsetRoot, FieldsetLegend
+githubLabel: 'component: fieldset'
+packageName: '@base_ui/react'
+---
+
+# Fieldset
+
+<p class="description">Fieldsets group multiple fields together with a label.</p>
```

Comment:
A sentence or two explaining what does this component offer on top of the native fieldset would be useful.